src/components/application_manager/test/mock_message_helper.cc

```diff
@@ -689,4 +689,6 @@ smart_objects::SmartObjectSPtr MessageHelper::CreateResponseMessageFromHmi(
   return MockMessageHelper::message_helper_mock()->CreateResponseMessageFromHmi(
       function_id, correlation_id, result_code);
 }
+void MessageHelper::RemoveEmptyMessageParams(
+    smart_objects::SmartObject& msg_params) {}
```

Comment:
I believe you miss something - for MessageHelper we need to add declaration of mocked method to mock_message_helper.h, and then do something like that:
```
return MockMessageHelper::message_helper_mock()->RemoveEmptyMessageParams(
     fmsg_params);
```
Please look to method above to get familiar with this concept

Reply:
done